KsApi/models/templates/LiveAuthTokenEnvelopeTemplates.swift

```diff
@@ -0,0 +1,5 @@
+extension LiveAuthTokenEnvelope {
+  internal static let template = LiveAuthTokenEnvelope(
+    token: "19b703c03de8efdf628694ce4c91a7bf8a6c98cfcbb7a089abbd488fdd166c4b"
```

Comment:
howzabout just using `"deadbeaf"` for the token. it's a nice way to discern testing stuff when we get output logging c.f. https://en.wikipedia.org/wiki/Hexspeak